### PR TITLE
Migrate samples to 1.0.9.

### PR DESCRIPTION
**Pull request by sergeybykov**
Updated samples to the API changes in 1.0.9.
Removed all dependencies on Orleans SDK (binaries and local silo environment).
Added a Host project to several samples that used to use SDK's local silo.

### CODE REVIEW

File: Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj [hunk 1]
```diff
@@ -67,9 +68,9 @@
     <Reference Include="Microsoft.WindowsAzure.Configuration">
       <HintPath>..\packages\Microsoft.WindowsAzure.ConfigurationManager.2.0.0.0\lib\net40\Microsoft.WindowsAzure.Configuration.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.WindowsAzure.ServiceRuntime">
-      <Private>True</Private>
-      <HintPath>..\packages\Unofficial.Microsoft.WindowsAzure.ServiceRuntime.2.4.0.0\lib\net40\Microsoft.WindowsAzure.ServiceRuntime.dll</HintPath>
+    <Reference Include="Microsoft.WindowsAzure.ServiceRuntime, Version=2.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\Program Files\Microsoft SDKs\Azure\.NET SDK\v2.5\bin\runtimes\base\Microsoft.WindowsAzure.ServiceRuntime.dll</HintPath>
```
Comment (gabikliot) [inline Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj]: not via nugget?

File: Samples/AzureWebSample/WebRole/WebRole.csproj [hunk 2]
```diff
@@ -52,8 +53,9 @@
       <HintPath>..\packages\Microsoft.Data.Services.Client.5.6.0\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.VisualBasic" />
-    <Reference Include="Microsoft.WindowsAzure.ServiceRuntime">
-      <HintPath>..\packages\Unofficial.Microsoft.WindowsAzure.ServiceRuntime.2.4.0.0\lib\net40\Microsoft.WindowsAzure.ServiceRuntime.dll</HintPath>
+    <Reference Include="Microsoft.WindowsAzure.ServiceRuntime, Version=2.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\Program Files\Microsoft SDKs\Azure\.NET SDK\v2.5\bin\runtimes\base\Microsoft.WindowsAzure.ServiceRuntime.dll</HintPath>
```
Comment (gabikliot) [inline Samples/AzureWebSample/WebRole/WebRole.csproj]: nugget?

File: Samples/Chirper/Host/Host.csproj [hunk 3]
```diff
@@ -28,65 +28,59 @@
...
-  <PropertyGroup>
-    <StartupObject>HelloWorldNuget.Program</StartupObject>
-  </PropertyGroup>
   <ItemGroup>
-    <Reference Include="CounterControl">
-      <HintPath>..\packages\Microsoft.Orleans.CounterControl.1.0.8\lib\net45\CounterControl.exe</HintPath>
-    </Reference>
     <Reference Include="Microsoft.Data.Edm">
-      <HintPath>..\packages\Microsoft.Data.Edm.5.6.0\lib\net40\Microsoft.Data.Edm.dll</HintPath>
+      <HintPath>$(SolutionDir)packages\Microsoft.Data.Edm.5.6.0\lib\net40\Microsoft.Data.Edm.dll</HintPath>
```
Comment (gabikliot) [inline Samples/Chirper/Host/Host.csproj]: why not via nugget?

File: Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj (hunk 1 above)
Comment (sergeybykov) [inline Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj]: This sample requires Azure SDK 2.5 anyway. And there is no official NuGet for Microsoft.WindowsAzure.ServiceRuntime.dll. So I think it's fine to reference it from the SDK.

File: Samples/AzureWebSample/WebRole/WebRole.csproj (hunk 2 above)
Comment (sergeybykov) [inline Samples/AzureWebSample/WebRole/WebRole.csproj]: Same here.

File: Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj (hunk 1 above)
Comment (gabikliot) [inline Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj]: Why not use 2.6?

Comment (sergeybykov) [inline Samples/AzureWebSample/OrleansAzureSilos/OrleansAzureSilos.csproj]: That wasn't my concern with this change. Easy to upgrade to 2.6.

**Comment by sergeybykov** [comment]
I think I'll merge this in for now, so that the samples compile and work with 1.0.9. There are important examples there of how to deal with the API changes and the removal of Local Silo. We can make any changes/improvement after that.

**Comment by gabikliot** [comment]
Sounds good.

**Pull request merged**